plugins/scaffolder-backend/api-report.md

```diff
@@ -61,13 +61,6 @@ export function createCatalogWriteAction(): TemplateAction<any>;
 // @public
 export function createDebugLogAction(): TemplateAction<any>;
 
-// @public (undocumented)
-export function createFetchCookiecutterAction(options: {
```

Comment:
did you mean to re-export this perhaps, so it stays around? the changeset says it shouldn't be breaking

Reply:
I thought about this, what do you think? Maybe it makes sense to rexport it for now, and then remove it when it's no longer shipped by default?